infrastructure/code/code_html.go

```diff
@@ -59,6 +60,45 @@ func getDataFlowHtml(issue snyk.CodeIssueData) string {
 	return dataFlowHtml
 }
 
+func getCodeDiffHtml(fix snyk.ExampleCommitFix) string {
+	linesHtml := ""
+	for _, commit := range fix.Lines {
+		linesHtml += fmt.Sprintf(`
+		<div class="example-line %s"><code>%s</code></div>`, commit.LineChange, commit.Line)
+	}
+	return linesHtml
+}
+
+func getTabsHtml(fixes []snyk.ExampleCommitFix) string {
+	tabsHtml := `<div class="tabs-nav">`
+
+	for i, fix := range fixes {
+		// Add the is-selected class to the first tab item only
+		// The IDE handles the tab switching with the is-selected class
```

Comment:
I guess the JavaScript for changing tabs was left for IntelliJ, to implement like we do for the Dataflow links in https://github.com/snyk/snyk-intellij-plugin/pull/497. 

I think that should be possible but since the interactions are all within the same panel and more "visual" based I wonder if it'd be better to include the JavaScript in the HTML. I could be wrong so feel free to challenge me! Also, if we end up deciding to put the interaction in the HTML I would leave that for a follow-up PR